dusty/cli/repos.py

```diff
@@ -9,9 +9,9 @@
 repo. To override a repo, use the `override` or `from` commands.
 
 Usage:
-  repos from <source_path>
+  repos from [--all] <source_path>
```

Comment:
what does `--all` mean when applied to `from`?  seems like this shouldn't be here